kyla_stearns/Fundamentals/basic13.rb

```diff
@@ -0,0 +1,89 @@
+# Print 1-255
+p (1..255).to_a
+
+# Print odd numbers between 1-255
+(1..255).to_a.select { |elem| p elem if elem.odd? } 
+
+# Print Sum
+def print_sum
+	sum = 0
+	for i in (1..255)
+		p "Number: #{i} and Sum: #{sum = sum + i}"
+	end
+end
+print_sum
+
+# Iterating through an array
+x = [1,3,5,7,9,13]
+x.each { |i| p i }
+
+# Find Max
+def find_max arr
+	p arr.max 
+end
+find_max [2,4,8,1,99]
+
+# Get Average
+def avg arr
+	# to_f will account for uneven division
+	p arr.reduce(:+).to_f / arr.size
```

Comment:
yas, like this reduce here. Great consideration on factoring in potential decimal values.